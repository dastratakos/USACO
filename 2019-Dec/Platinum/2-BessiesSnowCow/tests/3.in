100 200
64 39
88 39
9 39
66 39
58 66
77 88
50 58
2 58
6 58
99 2
86 6
27 86
94 27
13 94
90 94
85 94
74 90
71 90
17 85
20 17
35 17
81 35
19 35
60 71
24 60
38 17
59 60
82 19
98 19
47 82
34 47
55 98
29 82
97 47
75 97
32 29
79 75
84 97
44 84
62 97
48 75
53 48
76 44
69 44
18 76
42 18
41 69
28 42
63 76
30 42
80 28
67 80
11 67
70 63
15 11
83 15
36 80
72 83
93 70
95 93
100 72
10 100
1 72
16 1
73 16
49 73
96 95
89 96
14 96
52 73
54 14
5 14
33 52
56 14
68 54
46 33
7 68
78 56
8 46
22 33
23 56
21 52
4 21
3 21
43 8
12 3
87 12
31 87
61 3
37 31
25 37
65 61
26 37
57 61
51 26
92 57
45 21
91 45
40 45
1 42 40961
2 31
1 7 24430
2 3
1 21 70001
2 11
2 35
1 45 65830
2 32
2 29
1 82 93958
2 18
1 42 98912
2 48
2 20
1 56 38427
2 64
2 22
1 68 86671
1 3 21923
1 37 2027
2 93
1 83 95716
2 91
2 52
2 15
1 8 47438
2 2
1 20 38327
1 13 4958
2 75
1 11 47483
2 30
2 18
1 71 70561
2 83
1 1 18635
2 74
1 99 40344
2 72
2 11
1 95 49832
2 35
2 13
1 75 78940
1 75 71244
2 28
2 78
2 21
1 48 98463
2 9
2 93
2 49
1 20 14774
2 88
1 30 61165
1 5 88743
2 41
2 50
1 55 30140
2 22
2 56
2 12
1 47 42869
2 100
2 61
1 96 13690
1 27 91979
2 41
2 48
1 15 35363
1 1 16631
2 17
2 96
2 85
2 6
1 59 50248
1 85 93775
1 64 1234
1 34 10118
2 28
1 54 93825
1 49 26162
2 34
1 9 32752
2 98
1 28 18729
2 82
2 22
1 91 32798
1 96 51441
1 72 98697
2 98
1 68 39297
1 83 40620
1 11 33029
1 89 20013
2 19
2 94
1 67 39001
2 86
1 43 90863
1 19 99124
1 44 35471
1 81 48799
1 90 41025
1 2 25812
2 99
2 60
1 2 70865
1 47 16906
1 19 38949
2 55
1 82 7487
2 61
1 95 20466
1 50 97487
1 19 75811
1 24 96702
1 7 41579
2 7
2 13
1 37 18718
2 57
1 100 55996
1 68 51716
2 26
2 60
1 70 49089
2 98
2 49
2 90
2 46
2 73
1 15 60875
2 98
1 79 6145
2 68
1 91 58054
1 68 11927
1 62 14880
1 28 60541
1 34 65791
2 38
1 70 52126
2 97
2 72
2 6
1 20 80409
1 69 94519
2 12
1 20 30203
2 9
1 62 33900
1 33 33345
2 12
1 69 52559
2 89
1 85 3446
2 38
1 21 6369
2 86
2 30
1 51 58504
2 98
1 95 4825
1 5 69989
1 26 26038
1 47 51134
1 50 48305
2 30
1 14 48694
1 65 15615
2 99
2 40
2 35
2 27
2 24
2 91
2 38
2 66
1 25 98895
2 77
2 30
1 81 80540
1 5 11469
2 44
2 66
1 26 8509
2 88
1 76 70091
1 84 83485
2 51
1 78 93658
2 59
2 46
1 81 72457
1 82 37216
2 54
2 67
