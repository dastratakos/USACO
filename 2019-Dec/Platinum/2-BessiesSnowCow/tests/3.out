0
0
64
27
1
2
95
4
3
3
0
30
2
36
234
8
12
4
285
309
6
383
376
169
7
383
7
53
6
77
1
12
5
6
1
21
25
2
12
5
16
246
97
217
111
600
9
19
377
2
23
437
316
195
20
58
19
16
21
22
8
12
71
41
3
138
9
95
43
46
22
915
1388
272
36
25
36
21
22
308
15
45
15
29
5
705
347
28
7
23
158
26
15
1041
158
52
10
30
12
82
1580
